swap_meet/vendor.py

```diff
@@ -1,2 +1,67 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+
+    def __init__(self, inventory = None):
+
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):                         
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
+
+    def get_by_category(self, category):
+        return [item for item in self.inventory if category == item.category]
```

Comment:
Nice shorthand! Be sure to remove any extra comments. You can save extra comments locally but once you push you'd want to remove this